js/everything.js

```diff
@@ -281,6 +337,35 @@ var ZenPlayer = {
             toggleElement(event, ".plyr__video-wrapper", "Player");
         });
     },
+    setupAutoplayToggle: function()
+    {
+        // toggle auto next song playing
+        $("#toggleAutoplay").click(function(event)
+        {
+            var toggleTextElement = $("#" + event.currentTarget.id);
+            if (autoplayState === null)
```

Comment:
In that case, just use `autoplayState` as the condition. The same boolean rules will apply inside an if/loop condition